index.html

```diff
@@ -13,53 +13,183 @@
 
-    </footer>
-    
-    <div>
-        <ul>
-            <li><img class="ellipse" src="images/Ellipse 3.jpg" alt=""></li>
-            <li><img class="twitter" src="images/Vectortw.jpg" alt=""></li>
-            <li><img class="letteri" src="images/VectorI.png" alt=""></li>
-            <li><img class="lettern" src="images/Vectorn.jpg" alt=""></li>
-            <li><img class="letterm" src="images/Vectorm.jpg" alt=""></li>
-            <li><img class="git" src="images/Vectorgit.jpg" alt=""></li>
-            <li><img class="hand" src="images/Vectorhnad.jpg" alt=""></li>
-        </ul>
-    </div>
+    <section class="all4">
+        <div class="document4">
+            <img class="two" src="images/SnapshootPortfolio)1.jpg" alt="">
+            <h2 class="multipost"> Multi - Post
+                Stories
+            </h2>
+            <div class="list2">
+                <ul class="inline2">
+                    <li><img class="words2" src="images/Frame2.jpg" alt=""></li>
+                </ul>
+            </div>
+        </div>
+        <div class="part4">
+            <p class="subtitle1">A daily selection is privaly 
+                personalized reads; 
+                no accounts or sign-ups required
+            </p>
+            <div class="leng">
+                <ul class="point5">
+                    <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                    <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+                </ul>
+            </div>
+        </div>    
+    </section>
+
+    <section class="all5">
+        <div class="document5">
+            <h2 class="about"> About Myself </h2>
+            <img class="sub1" src="images/PrimaryText.jpg" alt="">
+            <h3 class="lets">LETS CONNECT</h3>
+        </div>
+        <div class="part5">
+            <div class="flex">
+                <ul class="point6">
+                    <li> <a href="https://github.com/luccamicale"><img class="git1" src="images/Vectorgit.jpg" alt=""></a></li>
+                    <li> <a href="#"><img class="letteri1" src="images/VectorI.png" alt=""></a></li>
+                    <li> <a href="#"><img class="lettern1" src="images/Vectorn.jpg" alt=""></a></li>
+                    <li> <a href="#"><img class="letterm1" src="images/Vectorm.jpg" alt=""></a></li>
+                    <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter1" src="images/Vectortw.jpg" alt=""></a></li>
+                    <li> <a href=""><img class="hand1" src="images/Vectorhnad.jpg" alt=""></a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="part6">
+            <div class="leng">
+                <ul class="point7">
+                    <li><a href="https://github.com/luccamicale"><img class="lenguage1" src="images/Enabledjj.jpg" alt=""></a></li>
+                    <li><img class="see1" src="images/Frame7.jpg" alt=""></li>
```

Comment:
- [ ] Kindly change these images into written HTML and CSS code, and kindly implement these changes to make your repo more professional. :100: